components/Document/DocumentViewer.tsx

```diff
@@ -510,6 +513,7 @@ const styles = StyleSheet.create({
   main: {
     position: "relative",
     justifyContent: "center",
+    width: "100%",
```

Comment:
@lightninglu10 Can you make sure this doesn't mess anything up with the document page in regular/full screen mode